### PR TITLE
Update runtime backend

### PR DESCRIPTION
**Pull request by carldebilly**
# Features

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [X] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

### CODE REVIEW

File: src/Uno.UI.Runtime.Skia.Wpf/WPF/WpfUIElementPointersSupport.cs [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+
+		private static PointerPointProperties BuildPointerProperties(MouseEventArgs args, int wheelDelta = 0)
+		{
+			var properties = new PointerPointProperties
+			{
+				IsLeftButtonPressed = args.LeftButton == MouseButtonState.Pressed,
+				IsRightButtonPressed = args.RightButton == MouseButtonState.Pressed,
+				IsPrimary = true
+			};
+
```
Comment (dr1rrb) [inline src/Uno.UI.Runtime.Skia.Wpf/WPF/WpfUIElementPointersSupport.cs:52]:
```suggestion
			properties.IsInRange = true;
			
```

File: src/Uno.UI.Runtime.Skia.Wpf/WPF/WpfUIElementPointersSupport.cs [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+				_ownerEvents.RaisePointerPressed(
+					new PointerEventArgs(
+						new Windows.UI.Input.PointerPoint(
+							frameId: GetNextFrameId(),
+							timestamp: (uint)args.Timestamp,
+							device: GetPointerDevice(args),
+							pointerId: 0,
+							rawPosition: new Windows.Foundation.Point(position.X, position.Y),
+							position: new Windows.Foundation.Point(position.X, position.Y),
+							isInContact: false,
```
Comment (dr1rrb) [inline src/Uno.UI.Runtime.Skia.Wpf/WPF/WpfUIElementPointersSupport.cs:168]:
```suggestion
							isInContact: true,
```

File: src/Uno.UI.Runtime.Skia.Wpf/WPF/WpfUIElementPointersSupport.cs [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+				_ownerEvents.RaisePointerMoved(
+					new PointerEventArgs(
+						new Windows.UI.Input.PointerPoint(
+							frameId: GetNextFrameId(),
+							timestamp: (uint)args.Timestamp,
+							device: GetPointerDevice(args),
+							pointerId: 0,
+							rawPosition: new Windows.Foundation.Point(position.X, position.Y),
+							position: new Windows.Foundation.Point(position.X, position.Y),
+							isInContact: false,
```
Comment (dr1rrb) [inline src/Uno.UI.Runtime.Skia.Wpf/WPF/WpfUIElementPointersSupport.cs:141]:
```suggestion
							isInContact: properties.HasPressedButton,
```

File: src/Uno.UI.Runtime.Skia.Wpf/WPF/WpfUIElementPointersSupport.cs [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+				_ownerEvents.RaisePointerExited(
+					new PointerEventArgs(
+						new Windows.UI.Input.PointerPoint(
+							frameId: GetNextFrameId(),
+							timestamp: (uint)args.Timestamp,
+							device: GetPointerDevice(args),
+							pointerId: 0,
+							rawPosition: new Windows.Foundation.Point(position.X, position.Y),
+							position: new Windows.Foundation.Point(position.X, position.Y),
+							isInContact: false,
```
Comment (dr1rrb) [inline src/Uno.UI.Runtime.Skia.Wpf/WPF/WpfUIElementPointersSupport.cs:114]:
```suggestion
							isInContact: properties.HasPressedButton,
```

File: src/Uno.UI.Runtime.Skia.Wpf/WPF/WpfUIElementPointersSupport.cs [hunk 5]
```diff
@@ -0,0 +1,234 @@
...
+				_ownerEvents.RaisePointerEntered(
+					new PointerEventArgs(
+						new Windows.UI.Input.PointerPoint(
+							frameId: GetNextFrameId(),
+							timestamp: (uint)args.Timestamp,
+							device: GetPointerDevice(args),
+							pointerId: 0,
+							rawPosition: new Windows.Foundation.Point(position.X, position.Y),
+							position: new Windows.Foundation.Point(position.X, position.Y),
+							isInContact: false,
```
Comment (dr1rrb) [inline src/Uno.UI.Runtime.Skia.Wpf/WPF/WpfUIElementPointersSupport.cs:87]:
```suggestion
							isInContact: properties.HasPressedButton,
```

File: src/Uno.UI.Runtime.Skia.Wpf/WPF/WpfUIElementPointersSupport.cs [hunk 6]
```diff
@@ -0,0 +1,234 @@
...
+				_ownerEvents.RaisePointerWheelChanged(
+					new PointerEventArgs(
+						new Windows.UI.Input.PointerPoint(
+							frameId: GetNextFrameId(),
+							timestamp: (uint)args.Timestamp,
+							device: GetPointerDevice(args),
+							pointerId: 0,
+							rawPosition: new Windows.Foundation.Point(position.X, position.Y),
+							position: new Windows.Foundation.Point(position.X, position.Y),
+							isInContact: false,
```
Comment (dr1rrb) [inline src/Uno.UI.Runtime.Skia.Wpf/WPF/WpfUIElementPointersSupport.cs:222]:
```suggestion
							isInContact: properties.HasPressedButton,
```

File: src/Uno.UWP/UI/Text/FontWeight.skia.cs [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+namespace Windows.UI.Text
+{
+	partial struct FontWeight
+	{
+		public SKFontStyleWeight ToSkiaWeight()
+		{
+			// Uno weight values are using the same system,
+			// so we can convert directly to Skia system
+			// without need for a mapping.
+			return (SKFontStyleWeight)this.Weight;
```
Comment (dr1rrb) [inline src/Uno.UWP/UI/Text/FontWeight.skia.cs:12]: If I'm right `Weight` is just an int so the value might be an invalid `SKFontStyleWeight`

Comment (carldebilly) [reply src/Uno.UWP/UI/Text/FontWeight.skia.cs:12]: Nope.  Because scalar values are valid too.  So that's correct.

**Pull request merged**